sql/log_event.cc

```diff
@@ -9068,8 +9068,10 @@ int Rows_log_event::next_record_scan(bool first_read) {
       if ((error = table->file->ha_index_read_map(
                table->record[0], m_key, HA_WHOLE_KEY, HA_READ_KEY_EXACT))) {
         DBUG_PRINT("info", ("no record matching the key found in the table"));
-        if (!is_trx_retryable_upon_engine_error(error))
+        if (!is_trx_retryable_upon_engine_error(error)) {
+          sleep(10);
```

Comment:
Is this intentional or just for debugging?

Reply:
It was just for debugging. Removed it.